src/kOS/Suffixed/StageValues.cs

```diff
@@ -1,28 +1,54 @@
 ﻿using System;
+using System.Linq;
 using kOS.Safe.Encapsulation;
-using kOS.Utilities;
+using kOS.Safe.Encapsulation.Suffixes;
 
 namespace kOS.Suffixed
 {
     public class StageValues : Structure
     {
-        private readonly Vessel vessel;
+        private readonly SharedObjects shared;
 
-        public StageValues(Vessel vessel)
+        public StageValues(SharedObjects shared)
         {
-            this.vessel = vessel;
+            this.shared = shared;
+
+            InitializeSuffixes();
+        }
+
+        private void InitializeSuffixes()
+        {
+            AddSuffix("RESOURCES", new Suffix<SuffixedList<ActiveResourceValue>>(GetResourceManifest));
```

Comment:
this is a new feature that is not reflected in the docs, I think ill add a few more to this class and make it its own structure in the docs. for now it is not that different from what we already have so im ok with just not mentioning it for now